lib/Data/RepositoryFormData.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @version //autogentag//
+ */
+
+namespace EzSystems\RepositoryForms\Data;
+
+/**
+ * Trait for repository data objects.
+ */
+trait RepositoryFormData
```

Comment:
Thanks. As much as I like _sleuth_, _freshness_, and the formality of `NoveltyAssessor`, I think I'll suggest the plain `NewnessChecker`.